app/src/main/java/org/simple/clinic/patient/recent/RecentPatientsView.kt

```diff
@@ -57,4 +81,13 @@ class RecentPatientsView(context: Context, attrs: AttributeSet) : FrameLayout(co
   fun clearRecentPatients() {
     groupAdapter.clear()
   }
+
+  private fun openPatientSummary(patientUuid: UUID) {
+    activity.screenRouter.push(
+        PatientSummaryScreenKey(
+            patientUuid = patientUuid,
+            caller = PatientSummaryCaller.SEARCH,
```

Comment:
Is this Caller valid? Aren't we opening this from home screen/recent patients?

Reply:
Maybe we need to add a new entry in `PatientSummaryCaller`.

Reply:
I thought about this but right now this `caller` has the same behavior as `SEARCH`.
The `caller` is used in `PatientSummaryScreen` to distinguish between a new and an existing patient.
@saket What do you think we should do here?

Reply:
You're right. We're going to see this problem when we open patient summary by scanning BP passports. We should consider renaming them to `NEW_PATIENT` and `EXISTING_PATIENT`.

Reply:
Yes I was thinking the same. Since `SEARCH` is not very meaningful here.
I will rename it then.

Reply:
We'll also require renaming test functions for patient summary. Please do that in a separate PR?

Reply:
I have already made this change in this PR.

Reply:
I see you've already renamed `SEARCH` to `EXISTING_PATIENT`. Don't forget to update test names in `PatientSummaryScreenControllerTest` and `PatientSummaryScreenKey#caller` to `type` or something.

Reply:
While you're at it, can you please also rename `PatientSummaryScreenKey#screenCreatedTimestamp` to `screenCreatedAt`?

Reply:
I could not find any test names specific to `caller`.

Reply:
> I see you've already renamed `SEARCH` to `EXISTING_PATIENT`. Don't forget to update test names in `PatientSummaryScreenControllerTest` and `PatientSummaryScreenKey#caller` to `type` or something.

Does that mean `PatientSummaryCaller` should be renamed as well?

Reply:
Yes. 

Let's do that in a later PR? We should get this PR merged first.